### ISSUE TITLE
stuck at logging in...

### ISSUE THREAD

**Issue by sz4bex** [dank/node-csgorc#17]
https://i.imgur.com/fa8NlIi.png

**Comment by chilledgamer1985** [comment]
yep something has changed with the steam loging  . i have not had anytime to check what yet tho

**Comment by chilledgamer1985** [comment]
go to node-csgorc-master\node-csgorc-master\node_modules\steam\lib and open servers.js change it for this one as its due to the steam  login server ip's are outdated  . this lot is the new ones 

`module.exports = [
// Washington, USA
{host: '155.133.229.250', port: 27017},
{host: '155.133.229.250', port: 27018},
{host: '155.133.229.250', port: 27019},
{host: '155.133.229.251', port: 27017},
{host: '155.133.229.251', port: 27018},
{host: '155.133.229.251', port: 27019},
{host: '155.133.230.50', port: 27017},
{host: '155.133.230.50', port: 27018},
{host: '155.133.230.50', port: 27019},
{host: '155.133.230.34', port: 27017},
{host: '155.133.230.34', port: 27018},
{host: '155.133.230.34', port: 27019},
{host: '155.133.254.132', port: 27017},
{host: '155.133.254.132', port: 27018},
{host: '155.133.254.132', port: 27019},
{host: '155.133.254.133', port: 27017},
// Virginia, USA
{host: '162.254.192.109', port: 27017},
{host: '162.254.192.109', port: 27018},
{host: '162.254.192.109', port: 27019},
{host: '162.254.192.108', port: 27017},
{host: '162.254.192.108', port: 27018},
{host: '162.254.192.108', port: 27019},
{host: '162.254.192.100', port: 27017},
{host: '162.254.192.100', port: 27018},
{host: '162.254.192.100', port: 27019},
{host: '162.254.192.101', port: 27017},
{host: '162.254.192.101', port: 27018},
{host: '162.254.192.101', port: 27019},
// Illinois, USA
{host: '162.254.193.46', port: 27017},
{host: '162.254.193.46', port: 27018},
{host: '162.254.193.46', port: 27019},
{host: '162.254.193.47', port: 27017},
{host: '162.254.193.47', port: 27018},
{host: '162.254.193.47', port: 27019},
{host: '162.254.193.6', port: 27017},
{host: '162.254.193.6', port: 27018},
{host: '162.254.193.6', port: 27019},
{host: '162.254.193.7', port: 27017},
{host: '162.254.193.7', port: 27018},
{host: '162.254.193.7', port: 27019},
// Frankfurt, Germany
{host: '162.254.197.40', port: 27017},
{host: '162.254.197.40', port: 27018},
{host: '162.254.197.40', port: 27019},
{host: '162.254.197.42', port: 27017},
{host: '162.254.197.42', port: 27018},
{host: '162.254.197.42', port: 27019},
{host: '162.254.197.180', port: 27017},
{host: '162.254.197.180', port: 27018},
{host: '162.254.197.180', port: 27019},
{host: '162.254.197.181', port: 27017},
{host: '162.254.197.181', port: 27018},
{host: '162.254.197.181', port: 27019},
// Amsterdam, Netherlands
{host: '155.133.248.53', port: 27017},
{host: '155.133.248.53', port: 27018},
{host: '155.133.248.53', port: 27019},
{host: '155.133.248.52', port: 27017},
{host: '155.133.248.52', port: 27018},
{host: '155.133.248.52', port: 27019},
{host: '155.133.248.50', port: 27017},
{host: '155.133.248.50', port: 27018},
{host: '155.133.248.50', port: 27019},
{host: '155.133.248.51', port: 27017},
{host: '155.133.248.51', port: 27018},
{host: '155.133.248.50', port: 27019},
// Madrid, Spain
{host: '155.133.246.69', port: 27017},
{host: '155.133.246.69', port: 27018},
{host: '155.133.246.69', port: 27019},
{host: '155.133.246.68', port: 27019},
{host: '155.133.246.68', port: 27019},
{host: '155.133.246.68', port: 27019},
// London, United Kingdom
{host: '146.66.152.11', port: 27017},
{host: '146.66.152.11', port: 27018},
{host: '146.66.152.11', port: 27019},
{host: '146.66.152.10', port: 27017},
{host: '146.66.152.10', port: 27018},
{host: '146.66.152.10', port: 27019},
{host: '162.254.196.83', port: 27017},
{host: '162.254.196.83', port: 27018},
{host: '162.254.196.83', port: 27019},
{host: '162.254.196.67', port: 27017},
{host: '162.254.196.67', port: 27018},
{host: '162.254.196.67', port: 27019},
{host: '162.254.196.68', port: 27017},
{host: '162.254.196.68', port: 27018},
{host: '162.254.196.68', port: 27019},
{host: '162.254.196.84', port: 27017},
{host: '162.254.196.84', port: 27018},
{host: '162.254.196.84', port: 27019},
// Stockholm, Sweden
{host: '155.133.242.9', port: 27017},
{host: '155.133.242.9', port: 27018},
{host: '155.133.242.9', port: 27019},
{host: '185.25.180.15', port: 27017},
{host: '185.25.180.15', port: 27018},
{host: '185.25.180.15', port: 27019},
{host: '155.133.242.8', port: 27017},
{host: '155.133.242.8', port: 27018},
{host: '155.133.242.8', port: 27019},
{host: '185.25.180.14', port: 27017},
{host: '185.25.180.14', port: 27018},
{host: '185.25.180.14', port: 27019},
];`

**Comment by emreires** [comment]
I'm still getting this problem.

**Comment by idontneedhelp** [comment]
Use the new servers from:
https://api.steampowered.com/ISteamDirectory/GetCMList/v1/?format=json&cellid=0
